pages/c/_cluster/istio/index.vue

```diff
@@ -31,7 +31,7 @@ export default {
     },
 
     monitoringUrl() {
-      return this.$route.fullPath.replace('/istio', '/monitoring');
+      return `/dashboard${ this.$route.fullPath.replace('/istio', '/monitoring') }`;
```

Comment:
`/dashboard` isn't always present (e.g. in dev) and shouldn't be hardcoded, nor should the product name..

```
import { NAME as MONITORING } from '@/config/product/monitoring';

...

{
          name:   'c-cluster-product',
          params: {
            cluster:  this.$route.params.cluster,
            product:  MONITORING
          }
}
```

Reply:
I didn't construct the link that way because I was having issues using a `nuxt-link` within a translation string. I've updated the page to use `router.push` when the a is clicked (or enter is pressed while it's focused)

Reply:
That works; you can use `router.resolve({ link object }).href` to get the computed path that a nuxt link will point to.

Reply:
That seems tidier; done